src/app/ubs-admin/components/ubs-admin-address-details/ubs-admin-address-details.component.html

```diff
@@ -1,43 +1,43 @@
 <div class="container mb-3">
   <div class="d-flex">
-    <h1 class="form__headline py-3">Адреса вивезення відходів</h1>
+    <h1 class="form__headline py-3">{{ "address-details.title" | translate }}</h1>
   </div>
   <form [formGroup]="addressDetailsForm">
     <div class="form-row">
       <div class="form-group col-md-6">
-        <label for="street">Вулиця</label>
+        <label for="street">{{ "address-details.street" | translate }}</label>
         <input type="text" formControlName="street" class="form-control" id="street" />
       </div>
       <div class="form-group col-md-5 offset-md-1">
-        <label for="addess-comment">Коментар до адреси від клієнта</label>
+        <label for="addess-comment">{{ "address-details.comment-label" | translate }}</label>
```

Comment:
Rename  'addess-comment' to 'address-comment'

Reply:
Done